test/parallel/test-fs-utimes-y2K38.js

```diff
@@ -7,17 +7,27 @@ const fs = require('fs');
 
 tmpdir.refresh();
 
-// Check for Y2K38 support. For Windows, assume it's there. Windows doesn't have
-// `touch`.
-if (!common.isWindows) {
+// Check for Y2K38 support. For Windows and AIX, assume it's there. Windows
+// doesn't have `touch` and `date -r` which are used in the check for support.
+// AIX lacks `date -r`.
```

Comment:
(Non-blocking observation.)
FWIW I believe GNU `date` is available via the `coreutils` package from the [AIX toolbox](https://www.ibm.com/support/pages/aix-toolbox-linux-applications-overview) but we don't currently have that installed on our CI hosts.

It is correct that the native `date` command doesn't support the `-r` option: https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/d_commands/date.html

Reply:
AIX doesn't natively have `stat` but does have `istat`: https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/i_commands/istat.html

Like GNU `date`, the GNU version of `stat` looks to be available in the `coreutils` AIX toolbox package.